repo2docker/buildpacks/__init__.py

```diff
@@ -2,6 +2,7 @@
 from .python import PythonBuildPack
 from .conda import CondaBuildPack
 from .julia import JuliaBuildPack
+from .julialegacy import JuliaLegacyBuildPack
```

Comment:
I'd import both from julia and deal with splitting them up there.
```suggestion
from .julia import JuliaLegacyBuildPack
```